src/quo2/foundations/resources.cljs

```diff
@@ -6,6 +6,16 @@
    :keycard-chip-dark  (js/require "../resources/images/ui2/keycard-chip-dark.png")
    :keycard-watermark  (js/require "../resources/images/ui2/keycard-watermark.png")})
 
+(def tokens
+  {:eth  (js/require "../resources/images/tokens/mainnet/ETH.png")
```

Comment:
hmm, perhaps we should use `tokens2` for consistency. All of the other "new" folders are marked with 2 while we migrate the codebase

Reply:
I am not sure which tokens are new and which are not. There are many tokens under the `tokens` directory and I guess we will still be using many of them.